src/adapters/sovrn.js

```diff
@@ -119,7 +119,7 @@ var SovrnAdapter = function SovrnAdapter() {
 
               //store bid response
               //bid status is good (indicating 1)
-              bid = bidfactory.createBid(1);
+              bid = bidfactory.createBid(1, bidObj);
```

Comment:
We are working towards this approach as a requirement for adapters but opted not to implement by core team as it became too obtrusive to modify adapter code ourselves.